Generate Java data models with javax validation constraints annotations (#428)
asyncapi/modelina: examples/README.md

- [asyncapi-from-object](./asyncapi-from-object) - A basic example where a AsyncAPI JS object is used to generate models.
- [asyncapi-from-parser](./asyncapi-from-parser) - A basic example where a AsyncAPI JS object from the [parser-js](https://github.com/asyncapi/parser-js) is used to generate models.
- [json-schema-draft7-from-object](./json-schema-draft7-from-object) - A basic example where a JSON Schema draft 7 JS object is used to generate models.

> - [java-generate-javax-constraint-annotation](./java-generate-javax-constraint-annotation) - A basic example where `javax.validation.constraints` annotations are used to generate Java models.

Reviewer: We are not generating Java models from `javax` but generating Java models with `javax` annotation 🙂 This change need to propagate to the rest of the docs 😄 

Think this is more fitting:
```suggestion 
- [java-generate-javax-constraint-annotation](./java-generate-javax-constraint-annotation) - A basic example where `javax.validation.constraints` annotations are included in the Java models.
``` 

What do you think @sudiptog81?

Author: Yes 😅
